updatecli/updatecli.d/charts/rating.yaml

```
@@ -0,0 +1,40 @@
+title: "Bump rating Helm Chart Version"
+
+scms:
+  default:
+    kind: github
+    spec:
+      user: "{{ .github.user }}"
+      email: "{{ .github.email }}"
+      owner: "{{ .github.owner }}"
+      repository: "{{ .github.repository }}"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      branch: "{{ .github.branch }}"
+
+sources:
+  lastChartVersion:
+    kind: helmchart
```

```suggestion
    kind: helmchart
    name: "Get the latest helm chart version"
```

missing "name" attribute as per the schema